src/main/resources/application.yaml

```diff
@@ -151,7 +151,9 @@ govnotify:
         email: 9c8ab848-b1f7-4dbe-9854-4922e94ece67
         sms: e805b3c5-b1e2-4ad1-9416-3298ca765738
       legalRep:
-        email: 33430912-89d5-416d-8d49-52eae9ce82c3
+        email:
+          ada: f43d0290-c78b-4698-a95e-b967cbd312b7
+          nonAda: 33430912-89d5-416d-8d49-52eae9ce82c3
```

Comment:
@augustio  the template 33430912-89d5-416d-8d49-52eae9ce82c3 doesn't match template per link in story - 

See the strike out of "Home Office" -
https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1191575812&__ncforminfo=suyFC86SYP_oSwJhuzaH6XmdScY6TxJcHuIEMzeZJu-Yl_XPFzP4B2whtcU7jtWOugP4CW0wR3PtLLy6WfFHv2RxZSb8hTNi&__ncforminfo=TeXtm3SEAq5_WgMNyXR3y72-5qTlZSYrc97AAOXtpLNpdyFCb899bop9TuADgTgaX3EkN_CG5PAu7r2gbbONKsR4MoEEHD35bxfwgUJompY=

Reply:
This is addressed in a different ticket.